README.md

```diff
@@ -3,9 +3,10 @@ Realtime status of the futel network.
 
 # running
 
-For local development:
+For local development you will need to clone [this repo](https://github.com/breedx2/futel-sqs-consumer)
 
 ```
+npm install /path/to/futel-sqs-consumer
```

Comment:
To clarify, the developer won't need to install `/path/to/futel-sqs-consumer` directly because it is referenced as a relative dir in `package.json`.  Maybe remove this part of clarify the above by showing both clones?

Reply:
updated... hopefully this makes sense.